lib/cartopy/crs.py

```diff
@@ -101,6 +97,19 @@ def x_limits(self):
     def y_limits(self):
         pass
 
+    @property
+    def threshold(self):
+        try:
```

Comment:
I'm a bit confused why you don't define `self._threshold` above? Then you shouldn't need the try/except.

Reply:
> I'm a bit confused why you don't define `self._threshold` above? Then you shouldn't need the try/except.

@greglucas Thank you for taking the time for reviewing my pull request! What do you mean exactly by 'above'? There seems no `__init__` method in `Projection` where I could define `self._threshold`.

Reply:
Can you add that method? Or perhaps a class-level attribute?

Reply:
Or you could replace this entire method with:
```python
@property
def threshold(self):
    return getattr('_threshold', 0.5)
```

Reply:
Using the `getattr` method sounds good.